doc/overview.xml

```diff
@@ -135,6 +135,14 @@ Groups library replaces the Gap 3 library of solvable groups of order at
 most 100. However, both the organisation and data descriptions of these
 groups has changed in the Small Groups library.
 <P/>
+
+<P>
+As of version 1.4 of this library, the arrangement of groups is the same as
+in Magma, Version 2.23. In earlier release the ordering in degrees
```

Comment:
Just noticed there’s something wrong in the language. “Earlier releases” maybe? (Although that implies ALL earlier releases; otherwise perhaps “some earlier releases”? Or specify which releases.

Reply:
All earlier releases is indeed what is meant. So we should indeed write "In earlier releases" or even "In all earlier releases of this library" (to avoid ambiguity with releases of Magma).

Reply:
Changed.